crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -298,17 +304,18 @@ impl SweetConductor {
         self.raw_handle().disable_app(id, reason).await
     }
 
-    /// Install the dna first.
-    /// This allows a big speed up when
-    /// installing many apps with the same dna
+    /// Adds the DnaFiles to the SweetConductor cache so that they can be re-added
+    /// to the RibosomeStore after a conductor restart.
+    /// The latter is required for the case of inline zomes since they are not
+    /// persisted to the wasm database and consequently don't automatically get
+    /// reloaded into the RibosomeStore after a conductor restart.
     #[cfg_attr(feature = "instrument", tracing::instrument(skip_all))]
-    async fn setup_app_1_register_dna(
+    async fn dna_files_to_sweet_conductor_cache(
         &mut self,
         dna_files: impl IntoIterator<Item = &DnaFile>,
     ) -> ConductorApiResult<()> {
         for dna_file in dna_files.into_iter() {
-            self.register_dna(dna_file.to_owned()).await?;
```

Comment:
This `register_dna()` was redundant. `DnaFile`s would always also get registered as part of app installation.